src/usr/local/emhttp/plugins/tailscale/include/Tailscale/System.php

```
@@ -66,6 +66,39 @@ public static function checkWebgui(Config $config, string $tailscale_ipv4): void
         }
     }
 
+    public static function checkServeConfig(): void
+    {
+        $ident_config = parse_ini_file("/boot/config/ident.cfg") ?: array();
+
+        $httpPort  = intval($ident_config['PORT'] ?? 80);
```

PHPStan (2.0.3)

Parameter #1 $value of function intval expects array|bool|float|int|resource|string|null, mixed given.


(argument.type)